app/src/main/java/org/oppia/app/application/ApplicationComponent.kt

```diff
@@ -29,6 +30,7 @@ import javax.inject.Singleton
   ApplicationModule::class, DispatcherModule::class, NetworkModule::class, LoggerModule::class,
   ContinueModule::class, FractionInputModule::class, ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
   NumberWithUnitsRuleModule::class, NumericInputRuleModule::class, TextInputRuleModule::class,
+  DragDropSortInputModule::class,
```

Comment:
For this & other modules, please wrap this so that you fit as many modules on one line as possible (within the 100 character limit). I don't think ktlint will handle formatting this correctly yet, so let's keep the code consistent in the meantime.

Reply:
will do

Reply:
ktlint is not enabled for app modules, but you can always run it on your file and I *think* it should just do it (need to test it though)

Reply:
i have used klint now @vinitamurthi can you please take a look